ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1947,3 +2009,32 @@ def _frontend_array_to_ivy(x):
         return x.ivy_array
     else:
         return x
+
+
+def args_to_frontend(
+        *args,
+        frontend,
+        include_derived=None,
+        **kwargs):
+    frontend_args = ivy.nested_map(args, _to_frontend(frontend), include_derived, shallow=False)
+    frontend_kwargs = ivy.nested_map(kwargs, _to_frontend(frontend), include_derived, shallow=False)
+    return frontend_args, frontend_kwargs
+
+
+def _to_frontend(frontend):
+    frontend_fn_dict = {"jax":
+                            ivy.functional.frontends.jax.numpy.array,
+                        "numpy":
+                            ivy.functional.frontends.numpy.array,
+                        "tensorflow":
+                            ivy.functional.frontends.tensorflow.constant,
+                        "torch":
+                            ivy.functional.frontends.torch.tensor}
```

Comment:
Let's make this more dynamic, we can create a private unified variable that points to the frontend array function, in that way we don't have to update the dict when a new frontend is added, thoughts? @fspyridakos